site/en/docs/privacy-sandbox/permissions-policy/index.md

```diff
@@ -0,0 +1,308 @@
+1. An iframe served from a cross-origin (`https://trusted-site.example`) that has been added to the origin list, and also has the `allow` attribute set on the iframe tag is allowed to use the feature.
+1. An iframe served from a cross-origin (`https://trusted-site.example`) that has been added to the origin list, but does not have the `allow` attribute set on the iframe tag is blocked from using the feature.
+1. An iframe served from a cross-origin (`https://ad.example`) that has not been added to the origin list is blocked from using the feature even with the `allow` attribute on the iframe tag.
+
+### HTTP Header
+
+{% Img src="image/hVf1flv5Jdag8OQKYqOcJgWUvtz1/jfhckpPdaepkw8bRPM0G.png", alt="HTTP response header diagram", width="800", height="459" %}
+
+```text
+Permissions-Policy: &lt;feature&gt;=(&lt;token&gt;|&lt;origin(s)&gt;)
+```
+
+A Permissions-Policy header in the response from the server is used to set the allowed origins for the feature. The header value can take a combination of tokens and strings of origins. The [available tokens](https://w3c.github.io/webappsec-permissions-policy/#structured-header-serialization) are `*` for all origins and `self` for same-origin.
+
+Example key-value pairs are: 
+* Syntax: `[FEATURE]=*` 
+  * Policy applied to all origins
+  * Example: `geolocation=*`
+* Syntax: `[FEATURE]=(self)`
+  * Policy applied to same-origin
+  * Example: `geolocation=(self)`
+* Syntax: `[FEATURE]=([ORIGIN(s)])` 
+  * Policy applied to the specified origins
+  * Example: `geolocation=("https://a.example" "https://b.example")`
+  * In addition to adding the origin to the origin list in the header, the iframe with that origin must set the allow attribute in the tag. The origin list header provides a list of valid origins for the allow attribute to consume. 
+* Syntax: `[FEATURE]=(self [ORIGIN(s)])`
+  * Policy applied to same origin and the specified origins
+  * Example: `geolocation=(self "https://a.example" "https://b.example")`
+* Syntax: `[FEATURE]=()` 
+  * Feature blocked for all origins
+  * Example: `geolocation=()`
+
+{% Aside 'gotchas' %}
+Separate multiple features in the header with a comma. Separate multiple origins in the origin list with a space between. 
+{% endAside %}
+
+{% Aside 'gotchas' %}
+If the Permission Policy header is not present in the response, the default value `*` token is used, which allows all iframes on the page with an `allow` attribute to use the feature. Therefore, it is strongly recommended that the origin list is explicitly set in the Permissions-Policy header to control access. 
+{% endAside %}
+
+{% Aside 'gotchas' %}
+With the change in Permissions Policy from Feature Policy, just adding the origin in the header origin list is no longer enough to enable the feature for a cross-origin iframe. The iframe must include the allow attribute if it’s a cross-origin regardless of what is set in the header origin list. 
+{% endAside %}
+
+### Iframe
```

Comment:
Since the heading before this is not an action, I think this should consistent with the previous one.  Let me know how "Iframe `allow` attribute" sounds.